src/components/SlateEditor/plugins/embed/SlateVideo.jsx

```diff
@@ -8,21 +8,45 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import Helmet from 'react-helmet';
+import defined from 'defined';
 import { Figure } from '@ndla/ui';
 import Button from '@ndla/button';
 import config from '../../../../config';
 import { EmbedShape } from '../../../../shapes';
 import EditVideo from './EditVideo';
 import DeleteButton from '../../../../components/DeleteButton';
+import * as visualElementApi from '../../../../containers/VisualElement/visualElementApi';
+
+const getIframeProps = ({ account, videoid, player = 'default' }, sources) => {
+  const source =
+    sources
+      .filter(s => s.width && s.height)
+      .sort((a, b) => a.height < b.height)[0] || {};
+  return {
+    src: `https://players.brightcove.net/${account}/${player}_default/index.html?videoId=${videoid}`,
+    height: defined(source.height, '480'),
```

Comment:
Hvorfor bruke defined i stedet for bare source.height || 'xx'? Viss den er 0 vil vi vel ikke bruk den uansett om den er definert